src/MailService.java

```diff
@@ -0,0 +1,27 @@
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+
+public class MailService<S> implements Consumer<Sendable<S>> {
+
+    private Map<String, List<S>> mailBox = new HashMap<String, List<S>>(){
+        @Override
+        public List<S> get(Object key) {
+            return super.getOrDefault(key,new LinkedList<>());
```

Comment:
```suggestion
            return super.getOrDefault(key, new LinkedList<>());
```
optional: read about Collections.emptyList()